### ISSUE TITLE
GTFS data is being processed. Please try again later

### ISSUE THREAD

**Issue by koeleboe** [iRail/iRail#473]
Since this morning, it is impossible to get the vehicle information. 
`{"error":503,"message":"Could not get data: GTFS data is being processed. Please try again later.. Please report this issue at https://github.com/irail/irail/issues/new"}`

**Comment by Bertware** [comment]
Yeah, NMBS had an invalid GTFS file yesterday, and this morning gtfs.irail.be became unreachable for some reason. Everything is up and running again now.

**Comment by koeleboe** [comment]
@Bertware The issue still persists with me! Still getting the same error

**Comment by koeleboe** [comment]
Maybe this is related to it, but going back in time again breaks the acces of the departures. Future works.

https://api.irail.be//liveboard/?id=BE.NMBS.008892007&station=Gent-Sint-Pieters&date=100323&time=0722&arrdep=departure&lang=en&format=json&alerts=false

https://api.irail.be//liveboard/?id=BE.NMBS.008892007&station=Gent-Sint-Pieters&date=100323&time=1422&arrdep=departure&lang=en&format=json&alerts=false
